File: solutions/csharp/isogram/1/Isogram.cs
using System;
using System.Linq;

public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        var clean = word.Where(Char.IsLetter);

        var count = clean.Select(Char.ToLower).ToHashSet().Count;
        return count == clean.Count();
    }
}

File: solutions/fsharp/isogram/1/Isogram.fs
module Isogram

open System 

let isIsogram (str: string) =
    let clean = str |> Seq.filter(Char.IsLetter)

    let uniqueCount =
        clean
        |> Seq.map (Char.ToLower)
        |> Set.ofSeq
        |> Set.count

    uniqueCount = (clean |> Seq.length)